static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -204,6 +232,52 @@ export class AccessMethod extends Component {
     })
   }
 
+  handleHarmonySelection(event, harmonyMethods) {
+    if (event !== '') {
+      this.setState({
+        selectValue: harmonyMethods.find(({ methodKey }) => methodKey === event).name
```

Comment:
a bit confused here can't we just destructure that value `methodKey` from the harmonyMethods then pull out the name from that?

```suggestion
        selectValue: harmonyMethods.find(({ methodKey }) => methodKey === event).name
```